todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailFragment.java

```diff
@@ -122,65 +188,161 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         super.onCreateOptionsMenu(menu, inflater);
     }
 
-    @Override
-    public void setLoadingIndicator(boolean active) {
-        if (active) {
-            mDetailTitle.setText("");
-            mDetailDescription.setText(getString(R.string.loading));
+    private void showTask(@NonNull Task task) {
```

Comment:
I consider that the job of the View is to display a task. How the view does this, by using the title, the description or anything else, is the View's choice and this should be tested by the UI tests. 
By exposing two different streams, one for the title and one for the description, you might end up with cases where the streams are not synchronised and I wanted to avoid this case. I know that in a sample project like this, it might not be the case, but I don't want other developers to take things very literal and end up [having problems like we did](https://upday.github.io/blog/mvvm_rx_common_mistakes).